README.md

```diff
@@ -42,12 +42,28 @@ client.quota.callback.static.kafka.admin.ssl.truststore.location=/tmp/trust.jks
 client.quota.callback.static.storage.per.volume.limit.min.available.bytes=5368709120
 
 # Check storage usage every 5 seconds
+# By default set to 60 seconds
 client.quota.callback.static.storage.check-interval=5
 
 # Optional list of principals not to be subjected to the quota
 client.quota.callback.static.excluded.principal.name.list=principal1,principal2
 ```
 
+### Properties and their defaults
+
+| Property                                                                  | Type    | Default                 | Description                                                                                                          |
+|---------------------------------------------------------------------------|---------|-------------------------|----------------------------------------------------------------------------------------------------------------------|
+| client.quota.callback.static.produce                                      | Double  | 1.7976931348623157e+308 | Produce bandwidth rate quota (in bytes)                                                                              |
+| client.quota.callback.static.fetch                                        | Double  | 1.7976931348623157e+308 | Consume bandwidth rate quota (in bytes)                                                                              |
+| client.quota.callback.static.request                                      | Double  | 1.7976931348623157e+308 | Request processing time quota (in seconds)                                                                           |
```

Comment:
How did we come to these defaults?

Reply:
these defaults and descriptions are all from `StaticQuotaConfig`
I didn't want to add there `Double.MAX_VALUE`, so I pasted the exact value here.

Maybe I should change it to `Double.MAX_VALUE`?

Reply:
I wonder why we use double there. But nothing we can change I guess.